tests/test/scheduler/test_mpi_world.cpp

```diff
@@ -63,22 +63,6 @@ TEST_CASE_METHOD(MpiBaseTestFixture, "Test creating world of size 1", "[mpi]")
     world.destroy();
 }
 
-TEST_CASE_METHOD(MpiTestFixture, "Test world loading from msg", "[mpi]")
```

Comment:
This was the only thing requiring `initialiseFromMsg` to take an extra boolean, and given that the function is implicitly tested all over the place, I thought it wasn't worth it.